lib/src/main/java/com/ibm/cloud/appid/android/internal/config/Config.java

```diff
@@ -63,41 +63,11 @@ public static String getPublicKeysEndpoint (AppID appId) {
 	public static String getIssuer(AppID appId) {
 
 		if (null != appId.overrideOAuthServerHost) {
-			return appId.overrideOAuthServerHost.split("/")[2];
+			String[] overrideServerUrlSplit =  appId.overrideOAuthServerHost.split("/");
+			return overrideServerUrlSplit[0] + "//" + overrideServerUrlSplit[2] + OAUTH_ENDPOINT + appId.getTenantId();
 		}
 
-		String region = appId.getBluemixRegion();
-		if (region == null) {
-			return serverUrlPrefix;
-		}
-
-		String issuer = region.contains("cloud.ibm.com") ? serverUrlPrefix + suffixFromRegion(region) :
-				Config.getOAuthServerUrl(appId);
-
-		return issuer.split("/")[2];
-	}
-
-	private static String suffixFromRegion(String region) {
```

Comment:
I think we will want to keep come sort of conversion from our old bluemix regions to new formats. One popular library we integrate with still uses the old branding

https://github.com/ibm-bluemix-mobile-services/bms-clientsdk-android-core/blob/master/lib/src/main/java/com/ibm/mobilefirstplatform/clientsdk/android/core/api/BMSClient.java